.circleci/config.yml

```diff
@@ -28,10 +40,32 @@ jobs:
           name: Test
           command: yarn test
 
+      - persist_to_workspace: *persist_work_dir
       - store_test_results:
           path: junit/
 
+  # Step release
+  release:
+    docker:
+      - image: cimg/node:14.16.1
+    working_directory: ~/repo
+    steps:
+      - attach_workspace: *attach_work_dir
+      - run:
+          name: Release
+          command: yarn semantic-release
```

Comment:
is it possible that this will change the version number to something else than what's committed?

Reply:
I'm not sure to get your question?

Reply:
I misunderstood what semantic-release does